src/components/buttons/Button/button.scss

```diff
@@ -0,0 +1,139 @@
+@import "../../../styles/variables";
```

Comment:
@georgiganchev-7DIGIT Please in all .scss files import only the theme file, as it contains everything needed (incl. all there css files).